frontends/web/src/routes/device/setup/success.jsx

```diff
@@ -42,9 +42,20 @@ export default class Success extends Component {
                             <Step title={t(`goal.step.4-${goal}.title`)} />
                         </Steps>
                         <hr />
-                        <h1 class={style.title}>{t(`success.${goal}.title`)}</h1>
-                        <div class={style.block}>
-                            {t(`success.${goal}.info`)}
+                        <h1 className={style.title} style="text-align: center;">
+                            {t(`success.${goal}.title`)}
+                        </h1>
+                        <div className={style.block}>
+                            <p style="text-align: center;">
+                                {t(`success.${goal}.summary`)}
+                            </p>
+                            { goal === 'create' ? (
+                                <ul class={style.summary}>
+                                    <li>{t(`success.${goal}.info1`)}</li>
```

Comment:
pls replace the $(goal) with 'create' here, as it can only be this, and it makes looking for keys easier.